src/security_utilities_rust/src/microsoft_security_utilities_core/identifiable_scans.rs

```diff
@@ -102,11 +102,12 @@ impl ScanMatch {
     }
 }
 
+#[derive(Clone)]
 pub struct PossibleScanMatch {
     name: &'static str,
     def_index: u32,
-    start: u64,
-    len: usize,
+    pub start: u64,
+    pub len: usize,
```

Comment:
It's preferable to have a start() and len() pub method vs exposing pub fields directly. Rust idiomatic approach is that, but sometimes I cut that corner.

Reply:
`start()` and `len()` was already there and I had missed it. Fixed: I now leverage these public methods instead.